test/upload_test_helpers.js

```diff
@@ -194,5 +194,9 @@ export default {
 
   getCloudfrontInvalidateOptions() {
     return s3Opts.cloudfrontInvalidateOptions
+  },
+
+  getIndexOptions() {
+    return s3Opts.indexOptions
```

Comment:
Simple functions that return a property of an object are a bit overkill.. it just creates delegation confusion I would recommend you not do this ever.